components/insight/SRC/org/openmicroscopy/shoola/env/rnd/RenderingControl.java

```diff
@@ -411,45 +411,36 @@ public void setActive(int w, boolean active)
      * managing the codomain transformations is rebuilt. 
      *
      * @param mapCtx The context to add.
-     * @param w The channel to add the context to.
-     * @throws RenderingServiceException If an error occurred while setting the value.
+     * @param index The channel to add the context to.
+     * @throws RenderingServiceException If an error occurred while adding the CodomainMapContext.
      * @throws DSOutOfServiceException If the connection is broken.
      */
-    public void addCodomainMap(CodomainMapContext mapCtx, int w)
+    public void addCodomainMap(CodomainMapContext mapCtx, int index)
             throws RenderingServiceException, DSOutOfServiceException;
     
     /**
-     * Updates the specified <code>CodomainMapContext</code>.
-     * The transformation associated should already be in the transformations.
-     * 
-     * @param mapCtx The context to update.
-     * @throws RenderingServiceException 	If an error occurred while setting 
-     * 										the value.
-     * @throws DSOutOfServiceException  	If the connection is broken.
-     */
-    //public void updateCodomainMap(CodomainMapContext mapCtx)
-    //	throws RenderingServiceException, DSOutOfServiceException;
-    
-    /**
      * Removed the <code>CodomainMapContext</code> from the list of
      * transformations.
      * 
      * @param mapCtx The context to remove.
-     * @param w The channel to remove the context from.
-     * @throws RenderingServiceException If an error occurred while setting the value.
+     * @param index The channel to remove the context from.
+     * @throws RenderingServiceException If an error occurred while removing the CodomainMapContext.
      * @throws DSOutOfServiceException If the connection is broken.
      */
-    public void removeCodomainMap(CodomainMapContext mapCtx, int w)
+    public void removeCodomainMap(CodomainMapContext mapCtx, int index)
             throws RenderingServiceException, DSOutOfServiceException;
 
     /**
      * Returns a read-only list of <code>CodomainMapContext</code>s using during
      * the mapping process in the device space.
      *
-     * @param w The channel
+     * @param index The channel
      * @return See above.
+     * @throws RenderingServiceException    If an error occurred while getting 
```

Comment:
Would it make sense to just say 'If an error occurred', the 'while ...' part basically just repeats what is said in the method description anyway?